requirements-dev.txt

```diff
@@ -21,3 +21,4 @@ bump2version==0.5.8
 # Test support
 matrix-synapse==1.5.1
 json5
+pytest-repeat
```

Comment:
was this added for flakes ones? i could not found any usage

Reply:
i haven't specified this in the PR description yet. it's a useful tool to repeat a test through the `pytest` command in the CLI

Reply:
I had some errors using this command before, i don't know what was the problem (if test syntax or just something that i was doing wrong) anyway it's a good way to discard flake tests yes.

Reply:
> I had some errors using this command before, i don't know what was the problem (if test syntax or just something that i was doing wrong) anyway it's a good way to discard flake tests yes.

syntax is `pytest <your_test_module_here.py> --count=n` where `n` is an integer